docs/quickstart-online.md

```
@@ -488,10 +488,25 @@ container, so no client is needed on the host:
 docker compose exec mongo mongosh --quiet \
-  "mongodb://mongo:27017/warehouse?directConnection=true" \
+  "mongodb://mongo:27017/views?directConnection=true" \
   --eval "db.order_state.countDocuments()"    # should reach 5
 ```
 
+To point a GUI or a driver at the store from your own machine instead, publish the
+port first — the stack does not, so nothing on the host can see it by default. Add
+`ports: ["127.0.0.1:27017:27017"]` to the `mongo` service in `docker-compose.yml`,
+re-run `docker compose up -d`, and connect with:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)